.github/workflows/connector_checklist.yml

```diff
@@ -0,0 +1,23 @@
+name: Add Connector Merge Checklist
+on:
+  pull_request:
+    types: [opened]
+    paths:
+      - "airbyte-integrations/connectors/source-**"
+      - "airbyte-integrations/connectors/destination-**"
+jobs:
+  checklist_job:
+    name: Add Connector Merge Checklist Job
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Checklist
+        uses: wyozi/contextual-qa-checklist-action@master
+        with:
+          comment-header: "### Before Merging a Connector Pull Request \n\n Wow! What a great pull request you have here! 🎉 \n\n To merge this Connector please ensure the following has been done/considered: \n\n"
+          comment-footer: "If the checklist is complete, but the CI check is failing, toggle the `checklist-action-run` on/off to re-run the checklist."
```

Comment:
Took a look at your demo, I couldn't see where one would "toggle the `checklist-action-run` on/off" - can you clarify?

Reply:
Ah its a label, will update to clarify

Reply:
ah! that probably moots my point about removing `labeled`/`unlabeled` then 😄

Reply:
... [indeed](https://github.com/airbytehq/airbyte/pull/25408#discussion_r1178358259) lol